index.js

```diff
@@ -63,7 +63,7 @@ module.exports.root = null;
 module.exports.register = register;
 
 function register() {
-  require.extensions['.jade'] = function(module, filename) {
+  require.extensions['.pug', '.jade'] = function(module, filename) {
```

Comment:
Documentation for `require.extensions` is sparse, but I believe this should work.